expr/agg.go

```diff
@@ -59,3 +59,27 @@ func (a *Aggregator) filter(rec *zed.Value) bool {
 	}
 	return !a.where(rec)
 }
+
+// AggregatorExpr is an expression variant of an Aggregator also referred to as
+// a streaming aggregation. They are stateful like Aggregations in a 'reduce'
+// context, but they return their current state after every input is consumed.
+func NewAggregatorExpr(agg *Aggregator) Evaluator {
```

Comment:
Reword this to describe NewAggregatorExpr or else s/AggregatorExpr/aggregatorExpr/ and then move the comment to that type.